app.js

```diff
@@ -7,10 +7,18 @@ require("./db/pg.js");
 
 // Logging
 const morgan = require("morgan");
+const router = require("./controllers/collect");
```

Comment:
I don't think we need this line

Reply:
oh weird I don't know where that came from.
Will push one more commit